com.unity.render-pipelines.core/Runtime/Debugging/FrameTiming/FrameTimeSample.cs

```diff
@@ -114,6 +119,8 @@ internal void DiscardOldSamples(int sampleHistorySize)
         {
             while (m_Samples.Count >= sampleHistorySize)
```

Comment:
I would check the sampleHistorySize to have a correct value. As I might call it with -1 and get an infinte loop.

Reply:
Good catch, I fixed this one right away.